interfaces/builtin/opengl.go

```diff
@@ -65,6 +65,15 @@ const openglConnectedPlugAppArmor = `
   /run/udev/data/c226:[0-9]* r,  # 226 drm
 `
 
+const openglConnectedPlugUdev = `
+# This file contains udev rules for opengl drivers.
+#
+# Do not edit this file, it will be overwritten on updates
+
+KERNEL=="card[0-9]*",         NAME="dri/card%%n", TAG+="%[1]s"
+KERNEL=="nvidia*|nvidiactl*", NAME="%%k",         TAG+="%[1]s"
```

Comment:
Drop 'NAME=' here and you missed /dev/vchiq.This should be:
```
SUBSYSTEM="drm", KERNEL=="card[0-9]*", TAG+="%[1]s"
KERNEL=="nvidia*", TAG+="%[1]s"
KERNEL=="vchiq", TAG+="%[1]s"
```

Since the apparmor policy has a redundancy too, can you update it to remove `/dev/nvidiactl rw,` and `/dev/nvidia-modeset rw,` since `/dev/nvidia* rw,` covers all of them?